diff --git a/node_modules/mongoose/types/query.d.ts b/node_modules/mongoose/types/query.d.ts
index 67129e6..2333ee9 100644
--- a/node_modules/mongoose/types/query.d.ts
+++ b/node_modules/mongoose/types/query.d.ts
@@ -117,7 +117,7 @@ declare module 'mongoose' {
     $where?: string | Function;
     /** @see https://www.mongodb.com/docs/manual/reference/operator/query/comment/#op._S_comment */
     $comment?: string;
-    $expr?: Record<string, any>;
+    $expr?: Record<string, any> | Record<keyof any, any> | boolean | any;
     // this will mark all unrecognized properties as any (including nested queries)
     [key: string]: any;
   };
